### DISCUSSION TITLE
Forgot password at SOC

### DISCUSSION THREAD

**Discussion by nicholaswkc34** [Security-Onion-Solutions/securityonion#12136] [General] [answered]
Dear forumer, how to reset password since i forgotten the password for Soc. Forgot email also.

**Comment by larry1788** [comment]
https://docs.securityonion.net/en/2.4/so-elastic-auth-password-reset.html
reset password

**Comment by reyesj2** [comment]
To find your email ssh to the manager and run

```sudo so-user list``` to change the password ```so-user password --email onionuser@somewhere.invalid```
You'll get a prompt to enter a new password then you can log back into SOC

**Reply by reyesj2** [reply, marked as answer]
The 2.4 docs were updated to show the steps outlined above. https://docs.securityonion.net/en/2.4/so-user.html
